docker/docker/src/test/java/org/arquillian/cube/docker/impl/client/containerobject/DockerContainerObjectBuilderTest.java

```diff
@@ -0,0 +1,499 @@
+package org.arquillian.cube.docker.impl.client.containerobject;
+
+import static org.hamcrest.CoreMatchers.*;
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.*;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.*;
```

Comment:
We tend to avoid start operator, I think that some IDEs do this automatically. No strong opinion here, so if you change ok if not is ok too.